python/ray/rllib/models/catalog.py

```diff
@@ -23,6 +27,11 @@
     "fcnet_hiddens",  # Number of hidden layers for fully connected net
     "free_log_std",  # Documented in ray.rllib.models.Model
     "channel_major",  # Pytorch conv requires images to be channel-major
+
+    # === Options for custom models ===
+    "custom_preprocessor",  # Name of a custom preprocessor to use
+    "custom_model",  # Name of a custom model to use
+    "custom_options",  # Extra options to pass to the custom classes
```

Comment:
Is this just a catch-all for extra options to pass to either the custom preprocessor or the custom model?

Reply:
Yeah, in case the user has any more